### DISCUSSION TITLE
How to format my assignmet for ease of reuse?

### DISCUSSION THREAD

**Discussion by Carl-N-Jarfalla** [community/community#147599] [GitHub Education]
### Hi everyone,

I'm quite new to github and having some trouble figuring out the best way to structure things.

For my assinments I often want to release weekly uppdates to the students introducing new parts of their project. But how do I do this so it's easy to then roll back the starter repository to "week 1" for the next course?

And a second question, is there an easy way to hide the students repositories after the assignment is done, at the moment they clutter up the list of repositories for the organization quite badly.

**Comment by V2U-Services** [comment]
Use consistent fonts, bullet points, and clear headings to format your assignment for easy reuse. Save templates for frequently used sections. Use version control and maintain the organization of your references. Ascertain appropriate spacing and use descriptive file names. This structure facilitates repurposing and future edits, saving time on related tasks.
